src/lxc/tools/lxc_start.c

```diff
@@ -231,6 +231,14 @@ int main(int argc, char *argv[])
 		exit(err);
 	lxc_log_options_no_override();
 
+	if (geteuid()) {
```

Comment:
Note that if running lxc-start nested inside an unprivileged container, geteuid() may not help.